enigma-principal/app/src/controller/km_http_server.rs

```diff
@@ -240,6 +221,14 @@ mod test {
     const REF_WORKER: [u8; 20] = [161, 186, 144, 238, 40, 242, 102, 161, 178, 93, 177, 83, 107, 128, 189, 132, 112, 8, 163, 252];
     const REF_CONTRACT_ADDR: [u8; 32] = [253, 20, 84, 186, 169, 51, 74, 146, 65, 95, 59, 133, 9, 25, 170, 193, 33, 159, 199, 204, 122, 116, 189, 122, 37, 132, 117, 188, 103, 120, 103, 137];
 
+    #[test]
+    fn test_foo() -> failure::Fallible<()> {
+        let foo: StateKeyRequest = serde_json::from_str("{\"data\": \"1234fF\", \"sig\": \"5678\"}").map_err(|e| {println!("{:?}", e); e})?;
+        println!("{:?}, {:?}", foo.sig, foo.data);
+
+        Ok(())
+    }
```

Comment:
This test always succeeds.
I think you can remove it, but maybe you should assert that the values of the `sig` and `data` fields are correct